org.js.nuclear.Nuclear.json

```diff
@@ -30,18 +30,15 @@
                 },
                 {
                     "type":"file",
-                    "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/org.js.nuclear.Nuclear.metainfo.xml",
                     "path": "org.js.nuclear.Nuclear.metainfo.xml"
                 },
                 {
                   "type":"file",
-                  "url":"https://raw.githubusercontent.com/advaithm/flathub/nuclear/org.js.nuclear.Nuclear.desktop",
                   "path": "org.js.nuclear.Nuclear.desktop"
                 },
                 {
                     "type":"file",
-                    "url":"https://github.com/advaithm/flathub/raw/nuclear/512.png",
-                    "dest-filename":"org.js.nuclear.Nuclear.png"
+                    "path":"org.js.nuclear.Nuclear.png"
```

Comment:
but the file in the repository is named "512.png" so this doesn't build.

Reply:
I'll rename it